routes/users.js

```diff
@@ -1,28 +1,28 @@
-const express = require('express');
-const passport = require('passport');
-const token = require('../utils/token'); 
-require('../config/facebook');
+const express = require("express");
+const authService = require("../service/Auth");
 
 // ToDo:// Store AuthToken in DB
 // Mark: Callback to generate access token
-function generateUserToken(req, res) {
-	console.log(`generating token for user ID ${req.user.id}`);
-	const accessToken = token.generateAccessToken(req.user.id);
-	console.log(`access Token: ${accessToken}`);
-	res.json(accessToken);
-}
 
 const router = express.Router();
 
-router.get('/auth/facebook', passport.authenticate('facebook', { scope: ['email'], session: false }));
+router.get("/auth", (req, res, next) => {
+  new authService(req.query.name).getAuthorize()(req, res, next);
```

Comment:
constructor name should not start with lower case

Reply:
this is not the constrictor it is the reference to the class which inherently call the constructor

Reply:
yep agree best practise is we should use PascalCase if we are using `new` keyword. You can change your constant name to "`AuthService`"

Reply:
It should be AuthService ( Starts with capital alphabet )